teku/src/main/java/tech/pegasys/teku/cli/options/Eth2NetworkOptions.java

```diff
@@ -169,6 +169,17 @@ public class Eth2NetworkOptions {
       hidden = true)
   private Boolean forkChoiceBeforeProposingEnabled = null;
 
+  @Option(
+      names = {"--Xfork-choice-first-descendent-as-head-enabled"},
```

Comment:
I'd be tempted to call this `--Xfork-choice-update-head-on-block-import-enabled` since we're totally skipping updating the chain head when importing a block. Sorry should have picked that up sooner to save a bunch of renaming...